app/gcl/associate-cloud-engineer/set-up-an-app-dev-environment-on-google-cloud/page.css

```
@@ -0,0 +1,282 @@
+.app-dev-environment-page .diagram { margin: 1.4rem 0; border: 1px solid var(--line); border-radius: 12px; background: var(--bg-2); padding: 1.2rem; overflow-x: auto; }
+.app-dev-environment-page .diagram__cap { font-family: var(--font-mono); font-size: 0.72rem; color: var(--muted-2); letter-spacing: 0.04em; margin: 0 0 0.7rem; }
+
+/* Mermaid sizing and alignment */
+.app-dev-environment-page .mermaid {
+  display: block;
+  width: 100%;
+}
+.app-dev-environment-page .mermaid > div {
+  width: 100%;
+}
+.app-dev-environment-page .mermaid :global(svg) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)